src/schema/artwork/contextGrids/index.ts

```diff
@@ -0,0 +1,163 @@
+import {
+  GraphQLInterfaceType,
+  GraphQLString,
+  GraphQLFieldConfig,
+  GraphQLList,
+} from "graphql"
+import { artworkConnection } from "schema/artwork"
+import { pageable } from "relay-cursor-paging"
+import { ResolverContext } from "types/graphql"
+import { first } from "lodash"
+import { convertConnectionArgsToGravityArgs } from "lib/helpers"
+
+interface GravityArgs {
+  exclude_ids?: string[]
+  page: number
+  size: number
+  total_count?: boolean
+  sort?: string
+  for_sale?: boolean
+  published?: boolean
+}
+
+export const formDefaultGravityArgs = ({ options, artwork }) => {
+  const { page, size, offset } = convertConnectionArgsToGravityArgs(options)
+  const gravityArgs: GravityArgs = {
+    page,
+    size,
+    exclude_ids: [artwork._id],
+  }
+
+  return {
+    gravityArgs,
+    offset,
+  }
+}
+
+const resolveType = data => {
+  return data.gridType
+}
+
+export const ContextGridType = new GraphQLInterfaceType({
+  name: "ContextGrid",
```

Comment:
This still sort of feels like a really generic-y name, that might be further scoped. Maybe `ArtworkContextGrid`?

Reply:
Yeah that works!